Formula/bash-preexec.rb

```diff
@@ -1,5 +1,5 @@
 class BashPreexec < Formula
-  desc "preexec and precmd functions for Bash just like Zsh."
```

Comment:
Enforcing the lack of full stops sounds good, but this one does seem to be an anomaly - running `ack "desc\b" *.rb | ack "\.$" *.rb` returned nothing else (though this branch is several days old by now).